src/main/resources/META-INF/plugin.xml

```diff
@@ -35,6 +35,8 @@
         <colorSettingsPage implementation="com.virtuslab.gitmachete.frontend.file.highlighting.MacheteColorSettingsPane"/>
 
         <annotator language="Machete" implementationClass="com.virtuslab.gitmachete.frontend.file.highlighting.MacheteAnnotator"/>
+
+        <errorHandler implementation="com.virtuslab.gitmachete.frontend.ui.impl.GitMacheteErrorHandler"/>
```

Comment:
Nice! And where/under what circumstances is it gonna show up to the user?

Reply:
whoops, forgive me for the poor description. Updated (see above)